### PR TITLE
Added Bubble Sort Algorithm and Happy Number in Rust

### PR DESCRIPTION
**Pull request by Yujin-nKim**
Hello, we are studying Rust language and want to contribute to your repo. This time, we implemented bubble sort in the Rust language and sent you a PR. Merge, please.😊

### CODE REVIEW

**Comment by Yujin-nKim** [comment]
@Marcosschwaab  @marlonchalegre  @tricinel  @melko  Hello, we left a PR. I hope to be Merge soon. I am interested in this repo.😊

**Comment by Yujin-nKim** [comment]
Hello. I additionally committed the code that implements the "happy number" algorithm in Rust. Please check and merge.

**Comment by Yujin-nKim** [comment]
Hello. I additionally committed the code that implements the "Insertion sort" and "selection sort" algorithm in Rust. Please check and merge. @Marcosschwaab @marlonchalegre @tricinel @melko

File: HappyNumber/HappyNumber.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+use std::io;
```
Comment (marlonchalegre) [inline HappyNumber/HappyNumber.rs:1]: Where this file is used in the bubble sort solution?

**Review by marlonchalegre — CHANGES_REQUESTED** [review]
Delete unnecessary files

**Pull request closed without merging**